Formula/h/harbor.rb

```diff
@@ -0,0 +1,20 @@
+class Harbor < Formula
+  desc "[Sandbox] Official Harbor CLI"
+  homepage "https://github.com/goharbor/harbor-cli"
+  url "https://github.com/goharbor/harbor-cli/archive/refs/tags/v0.0.1.tar.gz"
+  sha256 "8a764ea10173d8dca4c99c79f55b054bea184abf012e7a346b7b39fae094d805"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+    
+    ldflags = %W[
+      -s -w
+      -X github.com/goharbor/harbor-cli/cmd/harbor/internal/version.Version=#{version}
+    ]
+    system "go", "build", *std_go_args(ldflags:), "./cmd/harbor"    
+  end
+
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Added some test

Reply:
@SMillerDev